package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "djangogirls",
+  "private": true,
```

Comment:
this prevents this from accidentially being published. There's nothing secret, but this isn't really made for publishing; it's just a collection of dependencies